src/main/java/org/hisp/quick/batchhandler/AbstractBatchHandler.java

```diff
@@ -182,10 +182,12 @@ public final boolean addObject( T object )
     @Override
     public boolean insertObject( T object )
     {
-        final String sql =
+        String sql =
             statementBuilder.getInsertStatementOpening() +
             statementBuilder.getInsertStatementValues( object );
 

```

Comment:
Add check for sql.length() > 0 to avoid potential IndexOutOfBoundsException for negative index

Reply:
I instead return false if length is 0, since it means we are not adding anything. Let me know if > 0 is better here